--- /dev/null
+++ ../src/Twig/Runtime/RenderContentRuntime.php
@@ -25,7 +25,7 @@
 
     public function renderContent(ContentableInterface $contentableResource): string
     {
-        $content = html_entity_decode((string) $contentableResource->getContent(), \ENT_QUOTES);
+        $content = html_entity_decode((string) $contentableResource->getContent(), \ENT_NOQUOTES);
 
         return $this->contentParser->parse($content);
     }
